DDInstall/DDInstall.psm1

```diff
@@ -0,0 +1,192 @@
+function Install-DDAgent
+{
+    param (
+        [Parameter()][ValidateSet('DotNet')][String[]]$WithAPMTracers,
```

Comment:
note that I only started with support for .NET, but we can add other tracing libraries by simply adding other values to this set